src/test/resources/config.properties

```diff
@@ -0,0 +1,3 @@
+mainpage=http://automationpractice.com/index.php
+chromedriver=/Users/evgeniyamaslennikova/Desktop/chromedriver
```

Comment:
doesn't seem a good idea to put personal configuration to VCS ;) I won't be able to run this test